configuration/configuration.go

```diff
@@ -613,3 +619,8 @@ func (c *ConfigurationData) GetDevModePrivateKey() []byte {
 	}
 	return nil
 }
+
+// IsSaTokenEncrypted return pointer to bool
+func IsSaTokenEncrypted(encrypted bool) *bool {
```

Comment:
the name of this function is confusing: you just return a pointer to the given bool, don't you?

Reply:
ok, let me change that